File: tooling/sdkgen/test/typescript/openws-sdkgen-typescript-node/src/core/models/server/create-room-payload.ts
export interface CreateRoomPayloadInit {
    userId: string
    roomId: string
}

export class CreateRoomPayload implements CreateRoomPayloadInit {
    readonly userId: string
    readonly roomId: string

    constructor({ userId, roomId }: CreateRoomPayloadInit = {} as CreateRoomPayloadInit) {
        this.userId = userId
        this.roomId = roomId
    }
}

File: tooling/sdkgen/test/dotnet/UnityTestbed/Assets/Example.Chat.Sdk/Runtime/Example.Chat.Sdk/Core/Models/Server/CreateRoomPayload.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace Example.Chat.Core.Models.Server
{
    public partial class CreateRoomPayload
    {
        [JsonProperty("userId")]
        public string UserId;

        [JsonProperty("roomId")]
        public string RoomId;

    }
}